proto/prysm/v1alpha1/validator-client/web_api.proto

```diff
@@ -172,17 +172,19 @@ message CreateWalletRequest {
     string wallet_password = 2;
     // Mnemonic in case the user is creating a derived wallet.
     string mnemonic = 3;
+    // User specified mnemonic language.
+    string mnemonic_language = 4;
     // Number of accounts.
-    uint64 num_accounts = 4;
```

Comment:
Hi @saolyn we shouldn't be renumbering fields because this can break existing users with older versions. Instead, we should add the field as a new number, such as number 9 here